lib/src/converter/protobuf_converter.dart

```diff
@@ -61,10 +62,10 @@ class ProtobufConverter {
         ),
       );
 
-  Result<void, GenericFailure<ClearGattCacheError>> clearGattCacheResultFrom(
+  Result<Unit, GenericFailure<ClearGattCacheError>> clearGattCacheResultFrom(
```

Comment:
Now, this `Unit` type does have some exposure, because it's used in a declaration of a public method. Maybe in Dart it's okay to still not export the definition of this type, but that may cause some lesser or bigger issues on the library user side.

Reply:
I previously had it but since the public function is of type void there shouldn't be a problem in Dart. I think it could be a nice idea to change things to Unit later (this will break API ).

Reply:
Ah, so this one is not exposed? Then sure it's okay to keep `Unit` internal.